lib/tools/stats.js

```diff
@@ -0,0 +1,17 @@
+var convert = require('./convert_time');
+var BigInt = require('./bigint');
+
+module.exports = function(file) {
```

Comment:
Always name functions to help debugging :slightly_smiling_face: 

```suggestion
module.exports = function stat(file) {
```